### PR TITLE
Fixes to spec file to include the right release, source URL, man page, and docs.

### PR DESCRIPTION
**Pull request by sahanaprasad07**
Fixes to spec file to include the right release, source URL, man page and docs.
These errors were discovered by fedora-review
https://bugzilla.redhat.com/show_bug.cgi?id=2211754

### CODE REVIEW

File: packaging/pkcs11-provider.spec [hunk 1]
```diff
@@ -54,8 +55,8 @@ make check || if [ $? -ne 0 ]; then cat tests/*.log; exit 1; fi;
 
 %files
 %license COPYING
-%{_mandir}/man7/*
-%doc README
+%{_mandir}/man7/%{name}*
```
Comment (simo5) [inline packaging/pkcs11-provider.spec:58]: Why this change?
The man page will not match this
name = pkcs11-provider
manpage = provider-pkcs11.7

**Review by simo5 — CHANGES_REQUESTED** [review]

File: packaging/pkcs11-provider.spec (hunk 1 above)
Comment (sahanaprasad07) [reply packaging/pkcs11-provider.spec]: It  was a suggestion on the fedora-review bug that too unspecified wildcard was used.
If we don't want to align the manpage to name, then I guess we can let it be as is.
I have reverted the change.

**Review by simo5 — APPROVED** [review]

**Comment by simo5** [comment]
Thanks!

File: packaging/pkcs11-provider.spec [hunk 2]
```diff
@@ -24,6 +24,7 @@ BuildRequires: p11-kit-server
 BuildRequires: gnutls-utils
 BuildRequires: xz
 BuildRequires: expect
+BuildRequires: make
```
Comment (Jakuje) [inline packaging/pkcs11-provider.spec:27]: nit: I would move the `make` above the comment `# for tests`
